features/choose_javascript_database_strategy.feature

```diff
@@ -56,16 +56,16 @@ Feature: Choose javascript database strategy
       """
     And I write to "features/step_definitions/widget_steps.rb" with:
       """
-      Given /created? (\d) widgets/ do |num|
+      When(/^I create (\d+) widgets/) do |num|
         num.to_i.times { Widget.create! }
       end
 
-      Then /should have (\d) widgets/ do |num|
-        Widget.count.should == num.to_i
+      Then(/^I should have (\d+) widgets$/) do |num|
+        expect(Widget.count).to eq(num.to_i)
```

Comment:
This is good. I like the use of pronouns in steps - makes it more obvious that cucumber is about improving collaboration and makes it easier to adhere to business rules.

Reply:
The switch to `rspec-expectations` is good, too!